src/utils/storage.js

```diff
@@ -0,0 +1,6 @@
+export const COUNTER = "count";
+
+export const setLocalData = (key, value) =>
+	localStorage.setItem(key, JSON.stringify(value));
+
+export const getLocalData = key => localStorage.getItem(key);
```

Comment:
add json parse to this function itself as it is common in every usage and also add a alternative of {} to the output so that when the data doesnt exist or the item is null